packages/twenty-front/src/modules/apollo/utils/getImpersonationTokensFromStorage.ts.ts

```diff
@@ -0,0 +1,47 @@
+import { isDefined, parseJson } from 'twenty-shared/utils';
+import { type AuthTokenPair } from '~/generated/graphql';
+import { cookieStorage } from '~/utils/cookie-storage';
+
+export const getImpersonationTokensFromStorage = (): AuthTokenPair | undefined => {
```

Comment:
Feels a bit strange that this is called `getImpersonationTokensFromStorage` and not `getImpersonationTokens` while `getTokenPair` also gets since from storage but has not mention of storage